from .response import *

from piston.exception import RestException


class SaveObjectException(RestException):
    pass
